example/src/tabView/TabBarIconExample.tsx

```diff
@@ -13,57 +14,61 @@ import Contacts from './Shared/Contacts';
 
 type Route = {
   key: string;
+  title: string;
 };
 
 type State = NavigationState<Route>;
 
-export default class TabBarIconExample extends React.Component<{}, State> {
-  static title = 'Top tab bar with icons';
-  static backgroundColor = '#e91e63';
-  static appbarElevation = 0;
+export default function TabBarIconExample() {
+  const [index, setIndex] = React.useState(0);
+  const routes = [
```

Comment:
Same thing, let's move this outside of the component. You can also copy this from react navigation repo: https://github.com/react-navigation/react-navigation/tree/main/example/src/Screens/TabView